packages/website/src/components/PlaygroundPage/VersionedPlayground.tsx

```diff
@@ -0,0 +1,62 @@
+import { ComponentType, FC } from "react";
+import dynamic from "next/dynamic";
+import { useAvailableHeight } from "../../utils/useAvailableHeight";
+
+// Imported non-dynamically as a demo of version pinning.
+// 0.8.4 had `exports` configuration in its `package.json` which wasn't compatible with Next.js resolutions.
+// For this reason, support for 0.8.4 should be removed later (preferably before the PR is merged).
```

Comment:
Probably related: https://github.com/vercel/next.js/issues/46267

I had to replace `import` with `default` in components/package.json to fix it.